logging/bin/change_internal_password.sh

```diff
@@ -110,9 +112,9 @@ if [[ $response == 4* ]]; then
       ES_ADMIN_PASSWD=$(kubectl -n $LOG_NS get secret internal-user-admin -o=jsonpath="{.data.password}" |base64 --decode)
 
       # make sure hash utility is executable
-      kubectl -n $LOG_NS exec $targetpod --  chmod +x $toolsrootdir/tools/hash.sh
+      kubectl -n $LOG_NS exec $targetpod -c $targetcontainer --  chmod +x $toolsrootdir/tools/hash.sh
       # get hash of new password
-      hashed_passwd=$(kubectl -n $LOG_NS exec $targetpod --  $toolsrootdir/tools/hash.sh -p $NEW_PASSWD)
+      hashed_passwd=$(kubectl -n $LOG_NS exec $targetpod  -c $targetcontainer --  $toolsrootdir/tools/hash.sh -p $NEW_PASSWD|grep -v '*')
       rc=$?
```

Comment:
In OpenSearch 2.4.1, this utility now outputs a multi-line message indicating that it is deprecated and will be removed in the future prior to outputting the hashed password.  Thankfully, the message lines all start with a "#".